setup.py

```diff
@@ -52,5 +52,6 @@ def read(fname):
         "pywebpush>=1.13.0",
         "py-vapid>=1.7.0",
     ],
+    extras_require={"dev": ["black", "flake8", "isort", "mypy", "mypy-zope", "tox"]},
```

Comment:
It looks like our [https://github.com/matrix-org/sygnal/blob/master/tox.ini](`tox.ini`) file specifies some specific versions that it expects for these tools; could you please add those in here? (E.g., `"black"` → `"black==19.10b0"`)

Reply:
Great, I can make these changes sometime tonight. Thanks for the feedback! Will also incorporate the changes from the comment below.

Reply:
Okay all the requested changes have been made and the checks have passed so I hope this is good!